packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -558,8 +566,9 @@ class Navigator extends StatefulWidget {
   /// ```dart
   /// Navigator.pushNamed(context, '/nyc/1776');
   /// ```
-  static Future<dynamic> pushNamed(BuildContext context, String routeName) {
-    return Navigator.of(context).pushNamed(routeName);
+  static Future<dynamic> pushNamed(BuildContext context, String routeName,
+      {Map<String, dynamic> arguments}) {
```

Comment:
I meant spaces inside the named arguments, though yeah, putting this across multiple lines might be better. Just a nit though.